components/board/fake/board.go

```diff
@@ -418,6 +425,12 @@ func (s *DigitalInterruptWrapper) reset(conf board.DigitalInterruptConfig) error
 // Value returns the current value of the interrupt which is
 // based on the type of interrupt.
 func (s *DigitalInterruptWrapper) Value(ctx context.Context, extra map[string]interface{}) (int64, error) {
+	if err := s.Tick(ctx, true, 1); err != nil {
+		return 0, err
+	}
+	if err := s.Tick(ctx, false, 2); err != nil {
```

Comment:
The last argument to `Tick` is the timestamp, yeah? If you call `Value()` twice, we're going to have timestamps in the wrong order, which has the potential to be trouble. Add a `timestamp` field on the struct, use that here, and increment it every time you use it.

Reply:
it doesn't seem to be causing issues with different calls to Value. the RC card consistently increments. or do you mean something else?